### PR TITLE
feat(layout/#592): layout tabs - part 2 - `:tab*` commands and `gt/gT`

### PR DESCRIPTION
**Pull request by glennsl**
This wires up the `:tab*` commands and `gt/gT`.

Depends on https://github.com/onivim/libvim/pull/211
Addresses #592

### CODE REVIEW

File: src/Core/Utility/ListEx.re [hunk 1]
```diff
@@ -77,3 +77,75 @@ let findIndex = (predicate, list) => {
...
+/**
+ * removeAt
+ */
+let removeAt = (index, list) => {
+  let left = Base.List.take(list, index);
+  let right = Base.List.drop(list, index + 1);
+  left @ right;
+};
+
+let%test_module "removeAt" =
```
Comment (bryphe) [inline src/Core/Utility/ListEx.re:90]: Nice use of the inline tests! You turned me into a big fan of these.

File: src/Core/Utility/ListEx.re [hunk 2]
```diff
@@ -77,3 +77,75 @@ let findIndex = (predicate, list) => {
...
+
+/**
+ * insertAt
+ */
+let insertAt = (index, element, list) => {
+  let (left, right) = Base.List.split_n(list, index);
+  left @ [element] @ right;
+};
+
+let%test_module "removeAt" =
```
Comment (bryphe) [inline src/Core/Utility/ListEx.re:108]: `removeAt` should be `insertAt`?

Comment (glennsl) [reply src/Core/Utility/ListEx.re]: Oops. Good catch!

**Pull request merged**